sections/404-error.liquid

```diff
@@ -0,0 +1,61 @@
+
+<header class="section-header section-header--image has-parallax" style="background-image: url({{section.settings.background-404 | img_url: '1000x' }}); transform: translate3d(0px, 0px, 0px);" data-parallax="true" data-speed="0.5" data-direction="down">
+
+    <div class="text-wrapper--center-cell text-max-width">
+      <div class="text-wrapper--small text-center">
+        <h1 class="section-header__title text-center">{{ 'general.404.title' | t }}</h1>
+
+      </div>
+    </div>
+
+</header>
+<div class="grid error-message-wrapper">
+<div class="grid__item small--show medium--hide large--hide">
+  <span class="custom-title">{{ section.settings.errorTitle }}</span>
+<div class="line"></div>
```

Comment:
PUll the whole section-header structure over and we won't need any of these classes.  I'm talking about the code that's  in the 'if section.settings.show_collection_header' on the collection-template.  we shouldn't need to add margins or a one-off wrapper for this page, they will be picked up  by the global collection styles.